### PR TITLE
Zhian blog post 2023-11-29

### PR DESCRIPTION
**Pull request by zkamvar**
This is still in draft, I need to add images, but the text should be okay and ready to review.

I would like it if @ErinBecker, @froggleston, and @klbarnes20 could look it over to make sure it looks cromulent to them, but I also understand that this is a bit of a rush, so it's not terribly important.

### CODE REVIEW

File: _posts/2023/11/2023-11-27-sandpaper-0-15-0.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+starting on 2023-11-29, and it will automatically propogate to all lessons by
+2023-12-05.
+
+## Giving Thanks
+
+It has been over six months since we wrote a Dovetail blog post and needless to
+say, there are a lot of people that I would like to thank. I'm grateful for
+everyone who has taken the opportunity to open issues in The Workbench
+repositories to report bugs or features needed. I'm especially grateful for
+everyone who as additionally opened pull requests to fix these bugs.
```
Comment (klbarnes20) [inline _posts/2023/11/2023-11-27-sandpaper-0-15-0.md]:
```suggestion
everyone who has additionally opened pull requests to fix these bugs. 
```
I think this was meant to be has?

File: _posts/2023/11/2023-11-27-sandpaper-0-15-0.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+The whole reason this project works is because of the community. Full stop.
+Below, I list all the contributors from GitHub who have opened issues or pull
+requests that have lead to bugfixes or features in The Workbench, but I do want
+to acknowledge that there are so many more people who could be named here,
+including the Maintainers who have been able to give feedback directly in
+Maintainer meetings, Lesson Developers working in The Carpentries Incubator
+giving feedback about their use case, everyone from the internationalisation
+community whose continued efforts bring us closer to natively supporting
+translations, and last but not least, the brave souls who are willing to open
+and issue and dream that we will continue to grow.
```
Comment (klbarnes20) [inline _posts/2023/11/2023-11-27-sandpaper-0-15-0.md]:
```suggestion
an issue and dream that we will continue to grow.
```
is this supposed to be an?

**Review by klbarnes20 — APPROVED** [review]
Looks good to me!

File: _posts/2023/11/2023-11-27-sandpaper-0-15-0.md (hunk 1 above)
Comment (zkamvar) [reply _posts/2023/11/2023-11-27-sandpaper-0-15-0.md]: whoops!

**Review by froggleston — APPROVED** [review]
Looks great - thanks for writing this @zkamvar !

**Comment by ErinBecker** [comment]
It's a great post. Thank you @zkamvar. ❤️

**Comment by zkamvar** [comment]
I have added the images and it is now ready for your magic, @OscarSiba! Here's the preview link: https://deploy-preview-1738--stupefied-einstein-5bde90.netlify.app/blog/2023/11/sandpaper-0-15-0/

**Pull request merged**